packages/graphiql/src/components/GraphiQL.tsx

```diff
@@ -120,26 +123,28 @@ export type GraphiQLState = {
  */
 export const GraphiQL: React.FC<GraphiQLProps> = props => {
   if (!props.fetcher && !props.uri) {
-    throw Error('fetcher or uri property are required');
+    throw Error(i18n.t('Errors:GraphiQL requires a fetcher function'));
```

Comment:
this should be 'fetcher or uri property are required' for now. we can leave out that locale string from the `ru` translations for now until someone can give it another pass